rita_client_registration/src/lib.rs

```diff
@@ -0,0 +1,467 @@
+#![deny(unused_crate_dependencies)]
+use std::{
+    collections::{HashMap, HashSet},
+    sync::{Arc, RwLock},
+    thread,
+    time::{Duration, Instant},
+};
+
+use actix::System;
+use althea_types::{ExitClientIdentity, Identity, WgKey};
+use clarity::{
+    abi::{encode_call, AbiToken},
+    Address, PrivateKey, Uint256,
+};
+use futures::future::join_all;
+use phonenumber::PhoneNumber;
+use serde::{Deserialize, Serialize};
+use tokio::join;
+use web30::{
+    client::Web3,
+    jsonrpc::error::Web3Error,
+    types::{SendTxOption, TransactionResponse},
+};
+
+use crate::client_db::{
+    get_registered_client_using_ethkey, get_registered_client_using_meship,
+    get_registered_client_using_wgkey,
+};
+
+#[macro_use]
+extern crate log;
+#[macro_use]
+extern crate lazy_static;
+
+pub mod client_db;
+
+lazy_static! {
```

Comment:
this lazy static looks like logic that should be internal to the server logic. Maybe try and break it out as 'client registration server' and 'client registration lib' where Rita client for example only imports the lib.